chemprop/utils.py

```diff
@@ -273,23 +376,42 @@ def load_scalers(path: str) -> Tuple[StandardScaler, StandardScaler, StandardSca
     """
     state = torch.load(path, map_location=lambda storage, loc: storage)
 
-    scaler = StandardScaler(state['data_scaler']['means'],
-                            state['data_scaler']['stds']) if state['data_scaler'] is not None else None
-    features_scaler = StandardScaler(state['features_scaler']['means'],
-                                     state['features_scaler']['stds'],
-                                     replace_nan_token=0) if state['features_scaler'] is not None else None
+    scaler = (
+        StandardScaler(state["data_scaler"]["means"], state["data_scaler"]["stds"])
+        if state["data_scaler"] is not None
+        else None
+    )
```

Comment:
As above, I don't think breaking the inline if/else into multiline is very clear without other indent structure. If the line is long enough that it pushes formatting into that, then maybe we should just make them normally formatted if/else blocks instead of keeping the inline ordering.
```suggestion
    if state["data_scaler"] is not None:
        scaler = StandardScaler(state["data_scaler"]["means"], state["data_scaler"]["stds"])
    else:
        scaler = None
```

Reply:
i agree with this change. i simply used a code formatter, so it won't change the core structure of the code, but I think this block should not be a ternary statement